cmd/thanos/query.go

```diff
@@ -421,6 +421,9 @@ func runQuery(
 			router.Get("/", func(w http.ResponseWriter, r *http.Request) {
 				http.Redirect(w, r, webRoutePrefix, http.StatusFound)
```

Comment:
We can directly add the redirect to `webRoutePrefix+"/graph"` here so that we don't need to follow two redirects to get at the same place. Instead of `/` -> `/prefix` -> `/prefix/graph` directly redirecting `/` -> `/prefix/graph` would be better.

Reply:
:+1: